crates/humanode-runtime/src/lib.rs

```diff
@@ -391,13 +394,17 @@ impl pallet_authorship::Config for Runtime {
 
 parameter_types! {
     pub const TreasuryPotPalletId: PalletId = PalletId(*b"hmnd/tr1");
+    pub const EvmTreasuryPotPalletId: PalletId = PalletId(*b"hmnd/tr2");
     pub const FeesPotPalletId: PalletId = PalletId(*b"hmnd/fe1");
+    pub const EvmFeesPotPalletId: PalletId = PalletId(*b"hmnd/fe2");
```

Comment:
Similarily to https://github.com/humanode-network/humanode/pull/673/files#r1233230086, EVM does not need to have the fees pot